talk/app/talk-cli.hoon

```diff
@@ -382,20 +380,13 @@
       ~(tap by sessions)
     |-
     ?~  sez  [~ state]
-    =*  check-targets
-      %+  turn  ~(tap in viewing.session.i.sez)
-      ::  output warning if target is not subscribed
-      ::
+    =/  missing=(list target)
+      %+  murn  ~(tap in viewing.session.i.sez)
       |=  =target
-      ^-  _target
-      ?:  (subscription-check target)
-        target
-      =;  output=tape
-        ~&  output
-        target
-      %+  weld
-        "session {(scow %ta ses.sole-id.i.sez)} is not subscribed to "
-      ~(full tr target)
+      ?:  (subscription-check target)  ~
+      ~&  %+  weld  "not subscribed to {~(full tr target)} in "
+          "(scow %ta ses.sole-id.i.sez)}"
+      ~
```

Comment:
Well if we're not producing anything anyway, then a `turn` would've been fine too. iirc my suggestion was to use `murn` so that we may, after constructing the list, `~?` it if it's non-empty. Constructing a noun only to do nothing with it just feels awkward, is all.